src/App/Game/GameDarkSublayout.tsx

```diff
@@ -68,4 +79,40 @@ export class GameDarkSublayout extends React.Component<IGameDarkSublayoutProps,
       </div>
     );
   }
+  _getOptionsMenuLink() {
+    if (this.props.optionsMenuItems) {
+      return (
+        <Button
+          onClick={this._openOptionsMenu}
+          target="_blank"
+          variant="outlined"
+          style={{ margin: '8px', float: 'right' }}
+        >
+          <MoreVert style={{ color: 'white' }} />
+        </Button>
+      );
+    }
+  }
+
+  _openOptionsMenu = (event: any) => {
+    this.setState({ menuAnchorEl: event.currentTarget });
+  }
+
+  _closeOptionsMenu = (event: any) => {
+    this.setState({ menuAnchorEl: null });
+  }
+
+  _getOptionsMenu = () => {
+    const { menuAnchorEl } = this.state;
```

Comment:
you probably want to check that this.props.OptionsMenuItems exist, otherwise it doesnt make sense to return an empty menu